weblab_cg/tests/test_chaste_cg.py

```diff
@@ -0,0 +1,528 @@
+#
+# Tests the basics of weblab_cg
+#
+import cellmlmanip
+import logging
+import os
+import re
+import weblab_cg as cg
+import pytest
+import sympy
+from sympy import SympifyError
+import pyparsing
+
+# Show more logging output
+logging.getLogger().setLevel(logging.DEBUG)
+# TODO: Better docstrings
```

Comment:
Yes please!
These look like very complicated tests, much more complex than I'm used to seeing in docstrings (are there no simpler tests of small chunks we could be doing?)
Would like a lot of comments to document what these tests are supposed to do, so that we read the comments, then read the code and check if it does what the comments tell us it should be doing (we can only tell what the code _does_ by reading it, not what it should do / what the intention was)

Reply:
Yes at some point, for now I've split between a simple direct comparison test and my development tool test